plugin/minimal/minimal_test.go

```diff
@@ -0,0 +1,120 @@
+package minimal
+
+import (
+	"testing"
+
+	"github.com/coredns/coredns/plugin/test"
+	"github.com/miekg/dns"
+)
+
+type responseStruct struct {
```

Comment:
that test package has Case, i.e. `test.Case` that might be useful here as well.

Reply:
Okay I will take a look at that, I was following the general pattern in the plugins and examples I found

Reply:
@miekg I searched for `Case` in the default go testing pkg but i was unable to find it. Can you provide a link to the doc maybe?

Reply:
https://github.com/coredns/coredns/blob/master/plugin/test/helpers.go#L31

Reply:
Updated the test cases to use `test.Case` as suggested